en/docs/integrations/integration-concepts.md

```diff
@@ -1,57 +1,54 @@
   
- - **HTTP**: To interact with the HTTP API and communicate with an external endpoint via the HTTP protocol.   
- - **SMTP**: To send emails via the SMTP protocol.   
- - **POP3**: To receive emails from a POP3 email server.  
- - **IMAP**: To receive emails from any email server that supports the IMAP protocol. 
+ - **HTTP**: Communicates with external endpoints over HTTP.   
+ - **SMTP**: Performs email-related operations over SMTP. 
+ - **POP3**: Retrieves emails from a mail server over POP3. 
+ - **IMAP**: Retrieves emails from a mail server over a TCP/IP connection.
 
 Following are the connections you can use to interact with application-specific APIs:
 
- - **Twilio**: To interact with the Twilio API and perform operations such as sending messages, WhatsApp messages, and making voice calls.
- - **Slack**: To interact with the Slack API and perform required operations. 
- - **GitHub**: To interact with the GitHub API and  perform required operations. 
- - **Gmail**: To interact with the Gmail API and perform operations such as creating, modifying, and sending emails.
- - **Google Calendar**: To interact with the Google Calendar API and perform required operations.
- - **Google Sheets**: To interact with the Google Sheets API and perform operations such as creating and listing spreadsheets.
- - **Salesforce**: To interact with the Salesforce API and perform required operations. 
- - **NetSuite**: To interact with the NetSuite API and perform required operations. 
+ - **Twilio**: Integrate with Twilio to perform operations such as sending messages, WhatsApp messages, and making voice calls.
+ - **Slack**: Integrate with Slack to perform operations such as sending a message to a channel, deleting a message, etc.
+ - **GitHub**: Integrate with GitHub to perform operations such as creating an issue, creating a pull request, etc. 
+ - **Gmail**: Integrate with Gmail to perform operations such as sending an email, receiving an email, etc.
+ - **Google Calendar**: Integrate with Google Calendar to perform operations such as creating an event, deleting an event, etc.
+ - **Google Sheets**: Integrate with Google Sheets to perform operations such as reading data, formatting data, etc.
+ - **Salesforce**: Integrate with Salesforce to perform operations such as obtaining organization information, creating a record, etc.
+ - **NetSuite**: Integrate with Netsuite to perform operations such as creating a record instance, creating a sublist, etc. 
 
 ## Statement
-A statement in Choreo integrations is a syntactic unit that specifies the action to be carried out in an integration. 
 
-Choreo supports the following statements:
+A statement is a collection of code fragments that specifies the actions a Choreo integration will carry out. Choreo statements can also be composed of expressions. These expressions are written based on the [Ballerina Expression Syntax](#ballerina-expression-syntax).
```

Comment:
In Services page it says `A statement is a collection of code fragments that specifies the actions to be carried out by a ...` Shouldn't we follow consistent wordings for similar things?

Reply:
Is any of them confusing? Do they give out different meanings?

Reply:
Statements are same in integration or services. So no need to have difference in both pages.

Reply:
okay